tests/unit/test_dataset_converter.py

```diff
@@ -8,15 +8,35 @@
 import nwb2bids
 
 
-def test_dataset_converter_initialization(minimal_nwbfile_path: pathlib.Path, temporary_bids_directory: pathlib.Path):
-    dataset_converter = nwb2bids.DatasetConverter.from_nwb_directory(nwb_directory=minimal_nwbfile_path.parent)
+def test_dataset_converter_directory_initialization(
+    minimal_nwbfile_path: pathlib.Path, temporary_bids_directory: pathlib.Path
+):
+    nwb_paths = [minimal_nwbfile_path.parent]
+    dataset_converter = nwb2bids.DatasetConverter.from_nwb_paths(nwb_paths=nwb_paths)
+
     assert isinstance(dataset_converter, nwb2bids.DatasetConverter)
+    assert isinstance(dataset_converter.session_converters, list)
+    assert len(dataset_converter.session_converters) == 1
+    assert isinstance(dataset_converter.session_converters[0], nwb2bids.SessionConverter)
+
+
+def test_dataset_converter_file_paths_initialization(
```

Comment:
What about adding a test for testing `nwb_paths` containing both file and direcotry paths?